tasks/test_results_finisher.py

```diff
@@ -149,7 +149,7 @@ def process_impl_within_lock(
             db_session.add(totals)
             db_session.flush()
 
-        if self.check_if_no_success(previous_result):
+        if not self.check_if_any_success(previous_result):
```

Comment:
```suggestion
        if not any(previous_result):
```

The fn is now so simple you don’t need it anymore.